AY2021S2-CS2103-T14-1/tp
# Docs(PPP): Update Benedict's PPP

## What this does
This PR updates Community, Team Tasks and DG sections, and some links, of Benedict's PPP.

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Go to `About Us` then click on the `portfolio` link under Benedict's profile.
5. Check that the page is updated.
